spec/dish_spec.rb

```diff
@@ -4,6 +4,5 @@
 
   it { is_expected.to have_property :id }
   it { is_expected.to have_property :name }
-  it { is_expected.to belong_to :menu }
```

Comment:
Any reason why this test was removed? 🤔 According to your `dish` model definition, this is still true.